SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/CustomProperties.cs

```diff
@@ -109,31 +130,41 @@ public CustomProperties Clear(string key)
 
         private CustomProperties SetObject(string key, object value)
         {
-            if (IsValidKey(key))
+            if (ValidateKey(key))
             {
-                if (value != null)
+                if (value == null)
                 {
-                    Properties[key] = value;
+                    MobileCenterLog.Error(MobileCenterLog.LogTag, "Custom property value cannot be null, did you mean to call clear?");
```

Comment:
@guperrot Done. But you link to other message. [Same message](https://github.com/Microsoft/mobile-center-sdk-android/pull/437/files#diff-9f25f34f214d1a0aa8070633251140a8R159) doesn't have key.